### ISSUE TITLE
Plan/apply pipeline for new cluster

### ISSUE THREAD

**Issue by AntonyBishop** [ministryofjustice/cloud-platform#4105]
## Background

Create plan/apply pipeline for the new cluster

## Definition of done

- [ ] pipelines created and working

## Reference

[How to write good user stories](https://www.gov.uk/service-manual/agile-delivery/writing-user-stories)

**Comment by jackstockley89** [comment]
PR for concourse pipeline for the new cluster plan and apply 
https://github.com/ministryofjustice/cloud-platform-terraform-concourse/pull/152/files

**Comment by jackstockley89** [comment]
PR merged into main. Waiting on bootstrap pipeline to run, which is current on pause for live-2 cluster testing
